specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Trigger.json

```diff
@@ -365,6 +372,24 @@
           "maximum": 86400
         }
       }
+    },
+    "TumblingWindowDependency": {

```

Comment:
Are we planning on having conditional dependency for triggers? If not then it is better to name this as "TumblingWindowTriggerReference" to be consistent with other naming conventions.

Reply:
Resolved